test/check-networking

```diff
@@ -45,6 +45,7 @@ class TestNetworking(MachineCase):
             element = self.browser.text("#networking-interfaces tr:contains('%s')" % iface)
             for state in valid_states:
                 if state in element:
+                    self.browser.wait_visible("#networking-interfaces tr:contains('%s')" % iface)
                     return
             time.sleep(0.5)
```

Comment:
I"m making some changes so that we can wait on promises as well. This should solve a set of use cases, where you can send a javascript function to phantom that returns a promise ... and it'll just wait for it (or timeout).